searx/settings.yml

```
@@ -434,6 +434,23 @@ engines:
     engine: archlinux
     shortcut: al
 
+  - name: nixos wiki
+    engine: mediawiki
+    shortcut: nixw
+    base_url: https://wiki.nixos.org/
+    search_type: text
+    disabled: true
+    categories: [it, software wikis]
+
+  # Responded with 404 when implementing
+  # - name: nixos wiki unofficial
+  #   engine: mediawiki
+  #   shortcut: nixwu
+  #   base_url: https://nixos.wiki/
```

```suggestion
  #   base_url: https://nixos.wiki/
  api_path: api.php
```
This fixes nixos.wiki, see https://github.com/searxng/searxng/blob/94a0b415ef587e013df9e7350667b752a3822e90/searx/engines/mediawiki.py#L103

---

For more option see https://docs.searxng.org/dev/engines/mediawiki.html

---

You're completely right! New commit is up.